crates/fluvio-cluster/src/runtime/local/etcd.rs

```diff
@@ -0,0 +1,47 @@
+use std::{
+    fs::File,
+    path::PathBuf,
+    process::{Command, Stdio},
+};
+
+use fluvio_command::CommandExt;
+use tracing::info;
+
+use super::LocalRuntimeError;
+
+pub struct EtcdProcess {
+    pub log_dir: PathBuf,
+    pub data_dir: PathBuf,
+    pub base: PathBuf,
+}
+
+impl EtcdProcess {
+    pub fn start(&self) -> Result<String, LocalRuntimeError> {
+        let outputs = File::create(format!("{}/flv_etcd.log", self.log_dir.display()))?;
```

Comment:
separate out config file directory

Reply:
Not sure I understood what you mean here. This is a log file that will reside in the same directory as other logs. The directory itself is managed upper in the stack.